packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/Header/FaqIcon/index.js

```diff
@@ -11,7 +11,7 @@ class FaqIconContainer extends React.PureComponent {
     return (
       <FaqIcon
         highlighted={this.props.highlighted}
-        handleClick={() => this.props.actions.layoutWalletFaqClicked()}
+        handleClick={() => { this.props.actions.layoutWalletFaqClicked(); this.props.analytics.logClick('faq') }}
```

Comment:
Since this is an action, can we `take` it and submit the analytics action, instead of putting the analytics click in the template?

Reply:
Please consider making it a classes own method, and call it w/o creating new function on each render